### PR TITLE
ui edits

### PR DESCRIPTION
**Pull request by arsenkamavosyan**

### CODE REVIEW

File: app/src/main/java/com/dailyaims/screens/addtask/AddTaskFragment.kt [hunk 1]
```diff
@@ -32,6 +32,8 @@ class AddTaskFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+       val aimsModel= arguments?.getSerializable("Aims") as AimsModel?
```
Comment (artlazz) [inline app/src/main/java/com/dailyaims/screens/addtask/AddTaskFragment.kt]: replace cast to null type with null safe cast
"as AimsModel?" to "as? AimsModel"

File: app/src/main/java/com/dailyaims/screens/addtask/AddTaskFragment.kt [hunk 2]
```diff
@@ -32,6 +32,8 @@ class AddTaskFragment : Fragment() {
...
+        aimsModel
```
Comment (artlazz) [inline app/src/main/java/com/dailyaims/screens/addtask/AddTaskFragment.kt:35]: remove unused line

File: app/src/main/res/font/aims_font.xml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<font-family xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+    <font
+        android:font="@font/aims_font"
+        android:fontStyle="normal"
+        android:fontWeight="100"
```
Comment (artlazz) [inline app/src/main/res/font/aims_font.xml:7]: if I right fonts minimal weight is 400
check it please

File: app/src/main/res/font/aims_font.xml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+    <font
+        android:font="@font/aims_font"
+        android:fontStyle="normal"
+        android:fontWeight="400"
+        tools:ignore="ResourceCycle" />
+
+    <font
+        android:font="@font/aims_font"
+        android:fontStyle="normal"
```
Comment (artlazz) [inline app/src/main/res/font/aims_font.xml:18]: when font weight is 700 it is "bold" and not "normal"

File: app/src/main/res/layout/aims_layout.xml [hunk 5]
```diff
@@ -5,10 +5,10 @@
     android:id="@+id/card_view"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_margin="10dp"
+    android:layout_margin="8dp"
```
Comment (artlazz) [inline app/src/main/res/layout/aims_layout.xml:8]: You can create dimens file in res/values and save all dimens into the file, then use dimens from that file

File: app/src/main/res/font/aims_font.xml (hunk 3 above)
Comment (arsenkamavosyan) [reply app/src/main/res/font/aims_font.xml:7]: aper es xml file@ erevi jnjem , vorovhetev urish fonter em qashel

File: app/src/main/res/layout/aims_layout.xml (hunk 5 above)
Comment (arsenkamavosyan) [reply app/src/main/res/layout/aims_layout.xml:8]: ok knayem vonca arvum kanem

File: app/src/main/res/font/aims_font.xml (hunk 3 above)
Comment (artlazz) [reply app/src/main/res/font/aims_font.xml:7]: ok)

**Pull request merged**